sync3/handler/handler.go

```diff
@@ -280,20 +280,29 @@ func (h *SyncLiveHandler) setupConnection(req *http.Request, syncReq *sync3.Requ
 	var conn *sync3.Conn
 
 	// Identify the device
-	deviceID, accessToken, err := internal.HashedTokenFromRequest(req)
+	accessToken, err := internal.ExtractAccessToken(req)
 	if err != nil || accessToken == "" {
-		log.Warn().Err(err).Msg("failed to get device ID from request")
+		log.Warn().Err(err).Msg("failed to get access token from request")
 		return nil, &internal.HandlerError{
-			StatusCode: 400,
+			StatusCode: http.StatusUnauthorized,
 			Err:        err,
 		}
 	}
 
+	v2device, err := h.V2Store.DeviceByPlaintextAccessToken(accessToken)
+	if err != nil {
```

Comment:
We don't log `err` at all, which feels like a mistake? It would be nice to not log if it's just because its the first time we've seen the token (ErrNoRows)

Reply:
See 420d58a